### PR TITLE
Fix MDR fields so that it matches API v2 spec

### PR DESCRIPTION
**Pull request by yiumingtelnyx**

### CODE REVIEW

File: src/Telnyx.net/Entities/Messaging/Messaging_Sender_Ids/MessagingSenderId.cs [hunk 1]
```diff
@@ -4,6 +4,54 @@ namespace Telnyx
     using System.Runtime.Serialization;
     using Newtonsoft.Json;
 
+    public class MessagingFrom
```
Comment (thedonmon) [inline src/Telnyx.net/Entities/Messaging/Messaging_Sender_Ids/MessagingSenderId.cs:7]: I believe this should me MessagingTo according to these docs https://developers.telnyx.com/docs/api/v2/messaging/Messages

To and From should be switched as the From field is a string

Comment (thedonmon) [inline src/Telnyx.net/Entities/Messaging/Messaging_Sender_Ids/MessagingSenderId.cs:7]: `"to": [
      {
        "carrier": "T-MOBILE USA, INC.",
        "line_type": "Wireless",
        "phone_number": "+18665550001",
        "status": "queued"
      }
    ]`
Might be an array of these objects according to the docs

Comment (yiumingtelnyx) [reply src/Telnyx.net/Entities/Messaging/Messaging_Sender_Ids/MessagingSenderId.cs:7]: Yeah... the docs are wrong apparently or this needs to be a Union type.

Comment (yiumingtelnyx) [reply src/Telnyx.net/Entities/Messaging/Messaging_Sender_Ids/MessagingSenderId.cs:7]: Docs are wrong I think

**Comment by thedonmon** [comment]
Gotcha sorry to duplicate those comments

**Comment by d-telnyx** [comment]
@thedonmon / @yiumingtelnyx is this current ?

**Comment by thedonmon** [comment]
> @thedonmon / @yiumingtelnyx is this current ?

I don't believe so, need to double check the contracts for this object

**Pull request closed without merging**